media/js/firefox/whatsnew/whatsnew-133-experiment-eu.es6.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+import TrafficCop from '@mozmeao/trafficcop';
+import { isApprovedToRun } from '../../base/experiment-utils.es6.js';
+
+const href = window.location.href;
+
+const initTrafficCop = () => {
+    if (href.indexOf('v=') !== -1) {
+        if (href.indexOf('v=1') !== -1) {
+            // GA4
+            window.dataLayer.push({
```

Comment:
**Issue:** there's a JS error here as when this runs `window.dataLayer` has yet to be defined.

Suggest adding the following to each experiment JS file:

```javascript
if (typeof window.dataLayer === 'undefined') {
    window.dataLayer = [];
}
```